templates/export/resources.js

```diff
@@ -89,13 +89,88 @@ module.exports = Object.assign(
         "Principal": "apigateway.amazonaws.com"
       }
     },
+    "GenesysCodeVersion": {
+      "Type": "Custom::S3Version",
+      "Properties": {
+        "ServiceToken": {"Ref": "CFNLambda"},
+        "Bucket": {"Ref": "BootstrapBucket"},
+        "Key": {"Fn::Sub": "${BootstrapPrefix}/lambda/genesys.zip"},
+        "BuildDate": (new Date()).toISOString()
+      }
+    },
+    "GenesysLambda": {
+      "Type": "AWS::Lambda::Function",
+      "Properties": {
+        "Code": {
+          "S3Bucket": {"Ref": "BootstrapBucket"},
+          "S3Key": {"Fn::Sub": "${BootstrapPrefix}/lambda/genesys.zip"},
+          "S3ObjectVersion": {"Ref": "GenesysCodeVersion"}
+        },
+        "Environment": {
+          "Variables": {
+            outputBucket: {"Ref": "ExportBucket"},
+            s3Prefix: "connect/",
```

Comment:
It may be confusing to have Genesys files under connect S3 prefix. Should we put the files in `genesys/` s3 prefix instead?

Reply:
I changed the s3 prefix to Genesys (because it was copied from connect) -- however I don't see anywhere in the code that uses s3prefix for Genesys OR Connect...